src/COMPRESS/dump_atom_gz.cpp

```diff
@@ -15,8 +15,10 @@
 #include "domain.h"
 #include "error.h"
 #include "update.h"
+#include "force.h"
 
 #include <cstring>
+#include <fmt/format.h>
```

Comment:
format.h is not a system header but bundled with LAMMPS. This should be `#include "fmt/format.h"`